docs/source-pytorch/model/manual_optimization.rst

```diff
@@ -179,7 +179,7 @@ Learning Rate Scheduling
 
 Every optimizer you use can be paired with any
 `Learning Rate Scheduler <https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate>`_. Please see the
-documentation of :meth:`~pytorch_lightning.core.module.LightningModule.configure_optimizers` for all the available options
```

Comment:
This was correct, it used to work. We need to find the root cause of the issue.

Replacing these with the http links will show a link, but it will get outdated very quickly, as refactoring tools wont be able to take care of it.

Reply:
Ohk got it , will try to investigate it and fix it asap 👍